clang/tools/3c/3CStandalone.cpp

```diff
@@ -291,11 +292,11 @@ int main(int argc, const char **argv) {
   CcOptions.ConstraintOutputJson = OptConstraintOutputJson.getValue();
   CcOptions.StatsOutputJson = OptStatsOutputJson.getValue();
   CcOptions.WildPtrInfoJson = OptWildPtrInfoJson.getValue();
-  CcOptions.PerPtrInfoJson = OptPerPtrWILDInfoJson.getValue();
+  CcOptions.PerWildPtrInfoJson = OptPerPtrWILDInfoJson.getValue();
   CcOptions.AddCheckedRegions = OptAddCheckedRegions;
-  CcOptions.EnableAllTypes = OptAllTypes;
+  CcOptions.AllTypes = OptAllTypes;
   CcOptions.EnableCCTypeChecker = OptEnableCCTypeChecker;
-  CcOptions.WarnRootCause = OptWarnRootCause;
+  CcOptions.WarnRootCause = OptWarnRootCause || OptWarnAllRootCause;
```

Comment:
IMO, this kind of option canonicalization logic should stay in the `_3CInterface`.

Reply:
I didn't realize we had canonicalization of e.g. `_3COpts.BaseDir` happening inside `_3CInterface`, so, yes it makes sense for these to happen in the same place.